src/mon/MonCommands.h

```diff
@@ -1065,9 +1065,9 @@ COMMAND_WITH_FLAG("osd pool delete "
     FLAG(DEPRECATED))
 COMMAND("osd pool rm "
 	"name=pool,type=CephPoolname "
-	"name=pool2,type=CephPoolname,req=false "
-	"name=yes_i_really_really_mean_it,type=CephBool,req=false "
-	"name=yes_i_really_really_mean_it_not_faking,type=CephBool,req=false ",
+	"name=pool,type=CephPoolname "
+	"name=yes_i_really_really_mean_it,type=CephBool "
+	"name=yes_i_really_really_mean_it,type=CephBool ",
```

Comment:
i think each parameter has its own name in the sense that we can reference the value of the parameter by its name. see 

https://github.com/ceph/ceph/blob/08c722ef148ea7ac2247e7317e52c9afa721ce64/src/mon/OSDMonitor.cc#L12874-L12885

yes, the help message might be misleading. but this change breaks this functionality of this command =(

Reply:
Oops, thanks for pointing out
I've reverted changes to "yes_i_really_really_mean_it_not_faking"
I've made changes to pool name for 2nd time pool i/p
I'm confused as I couldn't find the reason for "yes_i_really_really_mean_it_not_faking"